project/Doc.scala

```diff
@@ -99,6 +99,8 @@ object UnidocRoot extends AutoPlugin {
     scalacOptions in (ScalaUnidoc, unidoc) ++= Seq("-skip-packages", "com.lightbend.lagom.internal"),
     javacOptions in doc := Seq(
       "-windowtitle", "Lagom Services API",
+      "-link", "https://doc.akka.io/japi/akka/current/",
+      "-link", "https://docs.oracle.com/javase/8/docs/api/",
```

Comment:
The current approach (https://github.com/lagom/lagom/pull/1283/files#diff-724cebf982c06b68575a9e9d2c6ff9a9R102) is wrong. It links to the Java API (see `CompletionStage`)  but won't link to Akka javadocs:

![screen_shot_2018-03-23_at_18_11_12](https://user-images.githubusercontent.com/762126/37843907-83f451d4-2ec6-11e8-83ce-6ec34762bab4.png)

According to `javadoc`'s manpages, `-link` must be followed by a URL where `packages-list` file exist (e.g. https://doc.akka.io/japi/akka/current/package-list). I only added the link to `docs.oracle.com` to test what was going on. Note that multiple `-link` arguments is supported and I also tested reordering the two `-link` so `docs.oracle.com` appeared first obtaining the same result.